node/core/dispute-coordinator/src/real/mod.rs

```diff
@@ -895,6 +895,32 @@ async fn issue_local_statement(
 			pending_confirmation,
 		)
 		.await?;
+		match rx.await {
+			Err(_) => {
+				tracing::error!(
```

Comment:
I suspect these will get noisy.

Metrics?

Reply:
actually no, those should not be noisy. The errors I would not expect to happen at all (hence errors) and the trace also only happens on `IssueLocalVote`, which should also not be noisy as this only happens on disputes and we will only vote once per dispute, so this should really be very low volume.